Sources/SwiftUICharts/BarChart/BarChartView.swift

```diff
@@ -40,7 +40,7 @@ public struct BarChartView : View {
         self.style = style
         self.darkModeStyle = style.darkModeStyle != nil ? style.darkModeStyle! : Styles.barChartStyleOrangeDark
         self.formSize = form!
-        self.dropShadow = dropShadow!
+        self.dropShadowColor = dropShadowColor!
```

Comment:
And probably you don't need this one as you have the `dropShadowColor` property inside style property and you don't use this as well later in the code